.readthedocs.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
+sphinx:
+  configuration: doc/conf.py
```

Comment:
Isnt this the wrong path? In beets we have docs/ 

sorry was just an as-is copy from other project!

Reply:
Yep, absolutely! No problem; it broke the build and I fixed it in the subsequent commit: ad73a61ec51c614c9d66e02202a532fc0caabb6d